FreeRTOS/Demo/RX600_RX64M_RSK_Renesas_e2studio/.cproject

```diff
@@ -151,5 +153,9 @@
 		<autodiscovery enabled="true" problemReportingEnabled="true" selectedProfileId=""/>
 	</storageModule>
 	<storageModule moduleId="org.eclipse.cdt.core.LanguageSettingsProviders"/>
-	<storageModule moduleId="refreshScope"/>
+	<storageModule moduleId="refreshScope" versionNumber="2">
+		<configuration configurationName="HardwareDebug">
+			<resource resourceType="PROJECT" workspacePath="/RTOSDemo"/>

```

Comment:
Not super familiar with this IDE, but this looks like it might be something specific to your personal workspace?

Reply:
I'll see if these changes can be omitted

Reply:
This change is one of usual behaviors of the e2 studio. I'm not familiar with other Eclipse based IDEs such as NXP's LPCXpresso and ST's SW4STM32 and so on. But Renesas's e2 studio do as follows.

(1) The e2 studio does not output 'perfect all' entries to the .cprojct file.
(2) I.e. only necessary (and something that e2 studio want to output) entries are output to the .cprojct file.
(3) When you change settings of your project, one or combination of the following changes in the .cproject file happen.
(3-1) Only the exact entries are changed.
(3-2) Only the exact entries are added.
(3-3) Sometimes entries which are unchanged but displayed in the same properties page are updated/added.
(3-4) Sometimes entries something like general purpos entries are updated/added.

This 'refreshScope' entry is the case (3-4).